Makefile

```diff
@@ -161,6 +173,20 @@ endif
 push-name:
 	@echo "pushed: $(IMAGE):$(VERSION)"
 
+manifest: .manifest-$(MULTIARCH_IMAGE) manifest-name
+.manifest-$(MULTIARCH_IMAGE):
+	@docker manifest create $(MULTIARCH_IMAGE):$(VERSION) \
```

Comment:
Looks like we need to set `DOCKER_CLI_EXPERIMENTAL=enabled` for any `docker manifest` subcommands to be accessible. I think it makes sense to do it inline here, each place the `docker manifest` cmd is used, rather than setting it more globally for the build.

Reply:
@skriss 
Something like this `export DOCKER_CLI_EXPERIMENTAL=enabled` is expected before every phase (2 time within and outside if) where manifest commands are used?

Reply:
you could change e.g. L178 to `@DOCKER_CLI_EXPERIMENTAL=enabled docker manifest create $(MULTIARCH_IMAGE):$(VERSION) \`, and similarly for subsequent lines

Reply:
@skriss 
Okay, you want me to test it out first? or if you already did, I will go ahead and push the changes

Reply:
I tried it and it worked properly for me, but additional testing wouldn't be a bad thing :)

Reply:
Done